Description: support setting BUILD_TIMESTAMP using SOURCE_DATE_EPOCH
 Enable reproducible builds by supporting setting the value of BUILD_TIMESTAMP
 through the SOURCE_DATE_EPOCH environment variable. More information at:
 https://reproducible-builds.org/specs/source-date-epoch/
Author: Jérémy Bobbio <lunar@debian.org>
Last-Update: 2016-01-23

--- libgcrypt20-1.6.4.orig/configure.ac
+++ libgcrypt20-1.6.4/configure.ac
@@ -1993,7 +1993,7 @@ changequote([,])dnl
 BUILD_FILEVERSION="${BUILD_FILEVERSION}mym4_revision_dec"
 AC_SUBST(BUILD_FILEVERSION)
 
-BUILD_TIMESTAMP=`date -u +%Y-%m-%dT%H:%M+0000 2>/dev/null || date`
+BUILD_TIMESTAMP=`date -d"@$SOURCE_DATE_EPOCH" -u +%Y-%m-%dT%H:%M+0000 2>/dev/null || date -u +%Y-%m-%dT%H:%M+0000 2>/dev/null || date`
 AC_SUBST(BUILD_TIMESTAMP)
 AC_DEFINE_UNQUOTED(BUILD_TIMESTAMP, "$BUILD_TIMESTAMP",
                    [The time this package was configured for a build])
